django_app/redbox_app/redbox_core/consumers.py

```diff
@@ -307,10 +307,10 @@ async def handle_citations(self, citations: list[AICitation]):
             for s in c.sources:
                 try:
                     file = await File.objects.aget(original_file=s.source)
-                    payload = {"url": str(file.url), "original_file_name": file.original_file_name}
```

Comment:
Don't we still want to use original_file_name here so we see 'stuff.pdf' in the frontend rather than 'me@cabinetoffice/stuff.pdf'?

Reply:
`file_name` will strip out `me@cabinetoffice/` and just return `stuff.pdf`

TBH i was hoping that we could just delete `` altogether in 30 days